dev-docs/bidders/bliink.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: BLIINK
+description: Prebid BLIINK Bidder Adaptor
+pbjs: true
+pbs: true
```

Comment:
I don't see a PBS pull request. Please set this to "false" or link to the PR.

Reply:
@bretg Fix

Reply:
I see, that changes are requested, but which ones ? @bretg thanks